internal/provider/kubernetes/kubernetes_test.go

```diff
@@ -53,7 +54,28 @@ func TestProvider(t *testing.T) {
 	svr, err := config.New()
 	require.NoError(t, err)
 	resources := new(message.ProviderResources)
-	provider, err := New(cliCfg, svr, resources)
+	ext := egcfgv1a1.Extension{
```

Comment:
can this be added aa a separate test ?

Reply:
removed a bunch of the extension manager passing around since #1208 handles the validation of HTTPRoutes/GRPCRoutes.